Makefile

```diff
@@ -46,7 +52,7 @@ ifdef OC_TOKEN
 	oc login ${OC_URL} --token=${OC_TOKEN}
 else
 	$(info **** Using OC_USER and OC_PASSWORD for login ****)
-	oc login ${OC_URL} -u ${OC_USER} -p ${OC_PASSWORD} --insecure-skip-tls-verify=true
+	oc login ${OC_URL} -u ${OC_USER} -p "${OC_PASSWORD}" --insecure-skip-tls-verify=true
```

Comment:
is it not working if it's not a string?

Reply:
I had a password containing a `)` and I get a syntax error

Reply:
ahhh nice, good catch then!